docs/install-and-run.md

```diff
@@ -263,40 +275,39 @@ your test and in your application code and use the built-in tools seamlessly.
 
 ### Swift
```

Comment:
Get rid of this section and move it closer to where are integrating w/ gem and manual installation when they're integrating with EarlGrey. Makes them optional for swift only.